packages/isc/coreutil/coreconst.go

```diff
@@ -6,36 +6,33 @@ import (
 
 // names of core contracts
 const (
-	CoreContractDefault         = "_default"
 	CoreContractRoot            = "root"
 	CoreContractAccounts        = "accounts"
 	CoreContractBlob            = "blob"
-	CoreContractEventlog        = "eventlog"
 	CoreContractBlocklog        = "blocklog"
-	CoreContractErrors          = "errors"
 	CoreContractGovernance      = "governance"
+	CoreContractErrors          = "errors"
+	CoreContractEvm             = "evm"
 	CoreEPRotateStateController = "rotateStateController"
 )
 
 var (
-	CoreContractDefaultHname         = isc.Hname(0)
 	CoreContractRootHname            = isc.Hn(CoreContractRoot)
 	CoreContractAccountsHname        = isc.Hn(CoreContractAccounts)
 	CoreContractBlobHname            = isc.Hn(CoreContractBlob)
-	CoreContractEventlogHname        = isc.Hn(CoreContractEventlog)
 	CoreContractBlocklogHname        = isc.Hn(CoreContractBlocklog)
-	CoreContractErrorsHname          = isc.Hn(CoreContractErrors)
 	CoreContractGovernanceHname      = isc.Hn(CoreContractGovernance)
+	CoreContractErrorsHname          = isc.Hn(CoreContractErrors)
+	CoreContractEvmHname             = isc.Hn(CoreContractEvm)
```

Comment:
```suggestion
	CoreContractEVMHname             = isc.Hn(CoreContractEVM)
```

Also, these EVM-related constants are unused? They should be referenced from package `evmnames` to avoid duplication (specifically the name `"evm"`).

Reply:
I thought the consts here should be reused in other places, but it seems few of them are reused in core contracts.